R/qenv-constructor.R

```diff
@@ -29,29 +27,28 @@ qenv <- function() {
 }
 
 
+#' Deprecated function
+#'
+#' The `new_qenv()` function is deprecated use [`qenv()`].
 #' @param code `r badge("deprecated")`
 #'  (`character(1)` or `language`) code to evaluate. Accepts and stores comments also.
 #' @param env `r badge("deprecated")` (`environment`)
 #'  Environment being a result of the `code` evaluation.
-#'
-#' @examples
-#' # create qenv with data and code (deprecated)
-#' new_qenv(env = list2env(list(a = 1)), code = quote(a <- 1))
-#' new_qenv(env = list2env(list(a = 1)), code = parse(text = "a <- 1", keep.source = TRUE))
-#' new_qenv(env = list2env(list(a = 1)), code = "a <- 1")
-#'
-#' @rdname qenv
 #' @aliases new_qenv,environment,expression-method
 #' @aliases new_qenv,environment,character-method
 #' @aliases new_qenv,environment,language-method
 #' @aliases new_qenv,environment,missing-method
 #' @aliases new_qenv,missing,missing-method
-#'
-#' @seealso [`base::within()`], [`get_var()`], [`get_env()`], [`get_warnings()`], [`join()`], [`concat()`]
+#' @return Errors.
+#' @seealso [`qenv()`]
 #'
 #' @export
 setGeneric("new_qenv", function(env = new.env(parent = parent.env(.GlobalEnv)), code = character()) {
-  lifecycle::deprecate_warn(when = " 0.5.0", what = "new_qenv()", with = "qenv()", always = TRUE)
+  lifecycle::deprecate_stop(
+    when = " 0.5.0",
+    what = "new_qenv()",
+    with = "qenv()"
+  )
   standardGeneric("new_qenv")
 })
 
```

Comment:
All below methods should be removed if you decided to go with `lifecycle::deprecate_stop(`.
But I would go one step further and even delete the generic